avocado/plugins/jobs.py

```diff
@@ -71,6 +104,38 @@ def handle_list_command(self, jobs_results):
 
         return exit_codes.AVOCADO_ALL_OK
 
+    def handle_show_command(self, config, jobs_results):
```

Comment:
I was expecting to be able to use the "references to job" that other commands, such as job replay accept.  For instance:
 *  `avocado run --replay=latest`
 * `avocado run --replay=~/avocado/job-results/job-2020-03-25T19.49-8974097`
 * `avocado run --replay=0c6935b5c863b2a`

All those should be possible and transparent if using `data_dir.get_job_results_dir()`, which is used in the previous commit.

Reply:
A *different* point that just occurred to me, is that, *maybe*, `avocado jobs show` could default to `latest`.  I think it would be a handy shortcut.

Reply:
I do have a few considerations when using multiple "things" to identify one object but since that the `get_job_results_dir()` is already implemented and being used, yes, makes sense.

About `avocado jobs show`, yes agree too. Will work on that.